README.md

```diff
@@ -5,6 +5,16 @@
 
 This repository contains the beta-stage publishing component of data.gov.uk.
 
+# Deployment
+
+Continuous Integration has been setup using github actions. 
+Tests are run on pull requests, deployments to Staging are via merges of branches into the `main` branch.
+In order to carry out a release to production a developer in the govuk team will need to create a release tag that leads with a `v` and approve of the deployment in github actions.
```

Comment:
I didn't understand "approve of the deployment in github actions" - I don't think GitHub Actions are involved in this step. I've rewritten what I think you meant below:

```suggestion
To deploy to Production, you'll need to [publish a Release](https://github.com/alphagov/datagovuk_publish/releases/new), specifying a "Tag version" beginning with `v` (e.g. `v1.0.0`).
```

Reply:
There is an additional step to approve a deployment, this is to protect secrets from leaking out as the `production` secrets cannot be protected with tagged releases.